sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
+* Small fix to the `offer_throughput` option in the async client's `create_database_if_not_exists` method. See [PR 32076](https://github.com/Azure/azure-sdk-for-python/pull/32076).
```

Comment:
what is the fix? Should we mention the kwargs name update?

Reply:
yep makes sense, should be clearer here